File: Python/TP1/TP2/exo2.py
import string 

def generator ():
    array=[]
    for i in string.ascii_lowercase :
        array.append(i)
    return array

def generator2 () :
    array=[]
    for i in string.ascii_lowercase :
        for a in range(2) :
            array.append(i)
    return array

def generatorN () :
    array=[]
    n = int(input("entrez un nombre"))
    for i in string.ascii_lowercase :
        for a in range(n) :
            array.append(i)
    return array

print(generatorN())

File: C/TP2/exo2.c
#include <stdio.h>

void table(int);

int main(){
    int a;
    printf("entrez un nombre : ");
    scanf("%d", &a);
    table(a);
}

void table(int a){
    for(int i =0;i<11;i++){
        int b;
        b=a*i;
        printf("%d x %d = %d\n",a,i,b);
    }
}